EIPS/eip-1571.md

```diff
@@ -0,0 +1,549 @@
+---
+eip: 1571
+title: EthereumStratum/2.0.0
+author: Andrea Lanfranchi [@AndreaLanfranchi](https://github.com/AndreaLanfranchi) <andrea.lanfranchi@gmail.com>, Pawel Bylica [@chfast](https://github.com/chfast) <pawel@ethereum.org>, Marius Van Der Wijden [@MariusVanDerWijden](https://github.com/MariusVanDerWijden)
```

Comment:
Change `[@AndreaLanfranchi](https://github.com/AndreaLanfranchi)` to `(@AndreaLanfranchi)` for all names.